src/list/index.tsx

```diff
@@ -362,15 +362,21 @@ export const List = factory(function List({
 				if (event.metaKey || event.ctrlKey) {
 					setActiveIndex(total - 1);
 				} else {
-					setActiveIndex((computedActiveIndex + 1) % total);
+					setActiveIndex(
+						(computedActiveIndex !== undefined ? computedActiveIndex + 1 : 0) % total
```

Comment:
Can we do this outside the function call? It's a bit messy to understand at the moment.